uaclient/tests/test_config.py

```diff
@@ -27,14 +27,25 @@ def test_accounts_returns_empty_list_when_no_cached_account_value(self):
 
         assert [] == cfg.accounts
 
-    def test_accounts_extracts_accounts_key_from_read_cache(self):
+    def test_accounts_extracts_accounts_key_from_account_read_cache(self):
         """Config.accounts property extracts the accounts key from cache."""
         tmp_dir = self.tmp_dir()
         cfg = UAConfig({'data_dir': tmp_dir})
         cfg.write_cache('accounts', {'accounts': ['acct1', 'acct2']})
 
         assert ['acct1', 'acct2'] == cfg.accounts
 
+    def test_accounts_extracts_accounts_key_from_machine_token_cache(self):
+        """Use machine_token cached accountInfo when no accounts cache."""
+        tmp_dir = self.tmp_dir()
+        cfg = UAConfig({'data_dir': tmp_dir})
+        accountInfo = {'id': '1', 'name': 'accountname'}
+
+        cfg.write_cache('machine-token',
+                {'machineTokenInfo': {'accountInfo': accountInfo}})
```

Comment:
uaclient/tests/test_config.py:45:17: E128 continuation line under-indented for visual indent

Reply:
fd05dced5aecb16bffaa09b5c5437f7d13f75a7afixed lint in commit